src/System.Private.ServiceModel/src/windows/project.json

```diff
@@ -32,11 +32,13 @@
     "System.Runtime.Extensions": "4.0.10",
     "System.Runtime.Handles": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
+	"System.Runtime.InteropServices.RuntimeInformation": "4.0.0-rc3-23725",
```

Comment:
Same observation -- is this package available for NET Native?  API Catalog says it isn't.  Also the indent on this line is different than the others